json_test.py

```diff
@@ -0,0 +1,75 @@
+import json
```

Comment:
What's this file for?

Reply:
I'm terrible for leaving bits of test scripts around in repos 😆  again thought I had added it to the `.gitignore`, going to blame git! (defo was me forgetting...)